### ISSUE TITLE
uniquery for evm indexer

### ISSUE THREAD

**Issue by roiLeo** [kodadot/uniquery#47]
make uniquery available for evm (kodadot/basick)

**Comment by vikiival** [comment]
It is available sir 

```
import { getClient } from '@kodadot1/uniquery'

const client = getClient('base')

const query = client.collectionListByName('harmony')

const res = await client.fetch(query)
console.log(res)
```

**Comment by roiLeo** [comment]
Thank you! Maybe indicate on the readme the list of available client?

**Comment by vikiival** [comment]
> Thank you! Maybe indicate on the readme the list of available client?

Yup I plan to `reexport Prefix from 'kodadot1/static'` so it is more user friendly

**Comment by vikiival** [comment]
added in #52
released in 0.7.0-rc.1

**Comment by vikiival** [comment]
published https://github.com/kodadot/uniquery/actions/runs/11181143417/job/31084457507
